entrypoint.sh

```diff
@@ -17,7 +17,7 @@ if [[ -f "build.gradle" ]]; then
   exit 1
 fi
 
-if [[ "${GITHUB_EVENT_NAME}" == "pull_request" ]]; then
+if [[ -z "${SCAN_PULL_REQUEST}" || "${GITHUB_EVENT_NAME}" == "pull_request" ]]; then
```

Comment:
I'm pretty sure it should be `&&` instead of `||` condition. Otherwise any additional commits to the PR are not analyzed.